pkg/api/oauthrelyingparty/provider.go

```diff
@@ -146,11 +146,17 @@ type Clock interface {
 	NowUTC() time.Time
 }
 
+type SimpleStore interface {
+	SetWithTTL(key string, value string, ttl time.Duration) error
+	GetDel(key string) (string, error)
```

Comment:
I suppose the implementation of this store will prefix the key with `app:{app_id}:` right? If that is the case, then it looks good to me.

Reply:
It would be `app:{app_id}:oauth:{provider_type}:`

Reply:
It would be better if it is `app:{app_id}:oauth:{provider_type}:{provider_alias}:`

Reply:
Sure